### PR TITLE
[tests-only]Add tests to check downloaded content

### LINKED ISSUE #39199: create acceptance tests for download.zip
Part of issue #39187 
Related to PR #39198 

1) check what acceptance tests we already have that select files/folders in the webUI and then click the "Download" button:
select-for-download

2) create/add scenarios so that we cover at least:
a) select 1 file and download it
b) select 1 folder and download it (the download should be `folderName.zip`)
c) select multiple files/folders and download (the download should be called `download.zip`)
d) select all files/folders and download (the download should be called `download.zip`)
e) inside a folder, do each of the above. For (c) and (d) the zip file should be named the same as the parent folder

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
 This PR adds acceptance tests steps to check the content of the downloaded folder. If we download multiple folders through UI it gets saved as `download.zip`. The added test steps check if the downloaded zip file contains all the folders or not by unzipping the zip file and checking the name of the folders/files inside it. In order to achieve this a `volume downloads` is mounted in the docker and it's mapped to the path where the folders actually get downloaded in selenium so that now the downloads will be saved to the `mounted volume`. This allows us to use the downloaded resources in the tests and check if everything is as expected.

## Related Issue

- Fixes : https://github.com/owncloud/core/issues/39199

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIFiles-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/33505/47/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIFiles-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/33506/47/1

File: .drone.star [hunk 1]
```diff
@@ -2683,7 +2702,7 @@ def setupScality(phpVersion, scalityS3):
         ] if createExtraBuckets else []),
     }]
 
-def fixPermissions(phpVersion, federatedServerNeeded, pathOfServerUnderTest = dir["server"]):
+def fixPermissions(phpVersion, federatedServerNeeded, selUserNeeded, pathOfServerUnderTest = dir["server"]):
```
Comment (dpakach) [inline .drone.star:2705]: you can give `False` as default for `sselUserNeeded` here to make it more cleaner

**Review by dpakach — COMMENTED** [review]
Looks good 👍🏼 
It would be nice have a test which checks the content of the file after unzipping from the downloaded file. Great work overall!

**Review by Talank — COMMENTED** [review]
LGTM other than already suggested changes.

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by sakshamgurung — COMMENTED** [review]
Very good

**Review by kiranparajuli589 — COMMENTED** [review]
looks good 👍🏼

**Review by phil-davis — APPROVED** [review]
LGTM - I made a couple of changes to step words (added "the" in a few places) and text of some exception messages. See the last commit.

Fantastic job! This is really good to be able to access the downloaded files. Next week we can think about looking inside the ZIP files and verifying that the actual expected real files are zipped in them.

**Pull request merged**